.circleci/config.yml

```diff
@@ -21,3 +21,6 @@ jobs:
           command: |
             /opt/conda/bin/flake8 --version
             /opt/conda/bin/flake8 obspy
```

Comment:
looks like if you don't make it another `- run` section it overwrites the above flake 8 check?

Reply:
Yup, that's right. I will fix that